.github/workflows/_linux_transformers.yml

```diff
@@ -45,6 +45,7 @@ permissions: read-all
 
 jobs:
   Torch-XPU-Transformers-Tests:
+    if: github.repository_owner == 'pytorch'
```

Comment:
Why are you adding this here? This workflow is intended to be run in torch-xpu-ops repository. With this change  workflow won't run at all which is wrong.

Reply:
Good catch! It is added by lint NO_WORKFLOWS_ON_FORK check auto correction, maybe we need to skip this checker. Let me roll back this change.

Reply:
Description is added. Thanks.